cmd/ttn-lw-cli/commands/use.go

```diff
@@ -78,6 +78,12 @@ var (
 					if err != nil {
 						return "", err
 					}
+					if _, err := os.Stat(dir); os.IsNotExist(err) {
+						if err = os.MkdirAll(dir, 0775); err != nil {
+							return "", err
+						}
+						logger.Infof("Created missing directory %s", dir)
+					}
```

Comment:
Let's just keep it simple:
```suggestion
					if err = os.MkdirAll(dir, 0775); err != nil {
						return "", err
					}
```

From https://golang.org/pkg/os/#MkdirAll
> If path is already a directory, MkdirAll does nothing and returns nil.